ajax/libs/chimee/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "chimee",
+  "description": "a video-player aims to bring wonderful experience on browser",
+  "filename": "index.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Chimeejs/chimee.git"
+  },
+  "keywords": [
+    "chimee",
+    "video",
+    "player",
+    "h5",
+    "hls",
+    "mp4",
+    "flv"
+  ],
+  "author": "toxic-johann",
+  "license": "MIT",
+  "homepage": "https://chimee.org/",
+  "npmName": "chimee",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```

Comment:
I thought the files `index.mjs` and `index.js` can't be used in web front-end.
So let's modify the pattern to let bot doesn't get this file.
Let's revise pattern from `**/*` to `index*.+(min.js|browser.js|css|map)`.
It's will helpful to us and developers. 😃 
Thank you.